g3web/WebContent/login.jsp

```diff
@@ -0,0 +1,12 @@
+<%@ taglib uri="/struts-tags" prefix="s"%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<s:form action="hello">

```

Comment:
@karthikvenkata @pranaychand  Action should not be hello. Make them close to real actions. For example this should be login.